src/components/Attachments/AttachmentCarousel/extractAttachmentsFromReport.js

```diff
@@ -35,7 +42,17 @@ function extractAttachmentsFromReport(parentReportAction, reportActions) {
             if (name === 'img' && attribs.src) {
                 const expensifySource = attribs[CONST.ATTACHMENT_SOURCE_ATTRIBUTE];
                 const source = tryResolveUrlFromApiRoot(expensifySource || attribs.src);
-                const fileName = attribs[CONST.ATTACHMENT_ORIGINAL_FILENAME_ATTRIBUTE] || FileUtils.getFileName(`${source}`);
+                if (uniqueSources.has(source)) {
+                    return;
+                }
+
+                uniqueSources.add(source);
+                let fileName = attribs[CONST.ATTACHMENT_ORIGINAL_FILENAME_ATTRIBUTE] || FileUtils.getFileName(`${source}`);
+
+                // image.jpg helps render the attachment as image if we didn't obtain an extension from the source
+                if (!/\.\w+$/.test(fileName)) {
```

Comment:
also asked this in slack but I wonder if it makes sense to make the network request to the URL and check the MIME type of the returned file to generate the default name.

Also wonder if "default name for unknown file" should be something that's implemented in `FileUtils.getFileName`

Reply:
NAB though, not sure if this is a problem per-se

Reply:
The code here is sync so it relies on identifying the file type by either attributes or the source itself. 
I didn't want to change the FileUtils logic as I don't fully understand the reasoning there, in the case of no extension it returns just the last portion of the URL, so we still have no extension to work with